apisix/plugins/slslog/rfc5424.lua

```diff
@@ -0,0 +1,107 @@
+--
```

Comment:
Is this file from https://github.com/detailyang/lua-resty-rfc5424/blob/master/lib/resty/rfc5424.lua?

Reply:
I reference it and change some codes for aliyun.